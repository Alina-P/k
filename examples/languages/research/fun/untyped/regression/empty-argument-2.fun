// testing empty argument

let x = 7
in let f() = x
   in f()

// 7
