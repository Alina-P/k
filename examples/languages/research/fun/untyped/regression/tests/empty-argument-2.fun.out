<result>
  7
</result>
